experiments/opencog/pattern_matcher_vqa/pattern_matcher_vqa.py

```diff
@@ -191,11 +223,13 @@ def get_answer(self):
         return self.attribute.name
 
 
-class OtherDetSubjObjConjunction(OtherDetSubjObj):
-    def __init__(self, strength, confidence, predicate_name):
+class ConjunctionResult(OtherDetSubjObj):
```

Comment:
See https://github.com/singnet/semantic-vision/pull/12/commits/7108ea6a64f1841931a8d1282785a0214dc3079c#r222984367 you can extract all values from resultData in the class,

Reply:
Done